### PR TITLE
User EnumZipper. Organize imports.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: common/src/main/scala/explore/AppMain.scala [hunk 1]
```diff
@@ -67,7 +68,7 @@ trait AppMain extends IOApp {
         Observation // TODO Remove this, it's here termporarily for testing URL automatic derivation.
           .Id(ProgramId.Science.fromString.getOption("GS-2020A-DS-1").get, Index.One)
           .some,
-      tabs = Zipper.fromNel(AppTab.all)
+      tabs = EnumZipper.of[AppTab]
```
Comment (cquiroz) [inline common/src/main/scala/explore/AppMain.scala:71]: Neat

File: common/src/main/scala/explore/model/reusability.scala [hunk 2]
```diff
@@ -3,29 +3,32 @@
...
-  implicit val expTargetReuse: Reusability[ExploreSiderealTarget]  = Reusability.derive
-  implicit val expObsReuse: Reusability[ExploreObservation]        = Reusability.derive
-  implicit def enumReuse[A: Enumerated]: Reusability[A]            =
+  implicit val statusReuse: Reusability[StreamingClientStatus]             = Reusability.derive
+  implicit val durationReuse: Reusability[Duration]                        = Reusability.by(_.getSeconds)
+  implicit val obsIdReuse: Reusability[Observation.Id]                     = Reusability.by(_.format)
+  implicit val siderealTargetReuse: Reusability[SiderealTarget]            = Reusability.byEq
+  implicit val expTargetReuse: Reusability[ExploreSiderealTarget]          = Reusability.derive
+  implicit val expObsReuse: Reusability[ExploreObservation]                = Reusability.derive
+  implicit def enumReuse[A: Enumerated]: Reusability[A]                    =
```
Comment (cquiroz) [inline common/src/main/scala/explore/model/reusability.scala:28]: Do we have a summon method to just do `Enumerated[A]`?

File: explore/src/main/scala/explore/Routing.scala [hunk 3]
```diff
@@ -27,16 +27,26 @@ object Routing {
...
       import dsl._
 
       (emptyRule
         | staticRoute(root, HomePage) ~> renderP(view => HomeComponent(view))
-        | staticRoute("/constraints", ConstraintsPage) ~> render(UnderConstruction())
         | dynamicRouteCT(("/obs" / string("[a-zA-Z0-9-]+")).pmapL(obsIdP)) ~> renderP(view =>
           HomeComponent(view)
-        ))
+        )
+        | dynamicRouteCT(("/target" / string("[a-zA-Z0-9-]+")).pmapL(targetObsIdP)) ~> renderP(
```
Comment (cquiroz) [inline explore/src/main/scala/explore/Routing.scala:45]: I don't recall if the order is important

File: explore/src/main/scala/explore/Routing.scala [hunk 4]
```diff
@@ -27,16 +27,26 @@ object Routing {
...
+          view => HomeComponent(view)
+        )
+        | staticRoute("/configurations", ConfigurationsPage) ~> render(UnderConstruction())
+        | staticRoute("/constraints", ConstraintsPage) ~> render(UnderConstruction()))
         .notFound(redirectToPage(HomePage)(SetRouteVia.HistoryPush))
         .verify(HomePage, ObsPage(Observation.Id.unsafeFromString("GS2020A-Q-1")))
```
Comment (cquiroz) [inline explore/src/main/scala/explore/Routing.scala:51]: Maybe we should verify `ConstraintsPage` and `ConfgurationsPage`

File: common/src/main/scala/explore/model/reusability.scala (hunk 2 above)
Comment (rpiaggio) [reply common/src/main/scala/explore/model/reusability.scala:28]: Good catch! Done.

File: explore/src/main/scala/explore/Routing.scala (hunk 3 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/Routing.scala:45]: I don't think it is. I'm just keeping the same order everywhere (`AppTab`, `RootModelRouting` and here) for mental sanity.

File: explore/src/main/scala/explore/Routing.scala (hunk 4 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/Routing.scala:51]: Yes, good idea! I'm adding that. And `TargetPage`.

**Pull request merged**